src/Makefile

```diff
@@ -11,7 +11,7 @@ else
 	FORCE_RPATH_STR=-Wl,-rpath=~$(FORCE_RPATH)~
 endif
 
-PREFIX?=/usr/local
+DESTDIR?=/usr/local/bin
```

Comment:
```suggestion
PREFIX?=/usr/local/bin
DESTDIR?=
```

Reply:
I propose to change this because IMHO because it's not really compliant with: https://www.gnu.org/prep/standards/html_node/DESTDIR.html

To be more specific:

> You should not set the value of DESTDIR in your Makefile at all; then the files are installed into their expected locations by default.

IMHO, `DESTDIR` must be empty by default. You don't need it for a standard install. It's a feature for "packagers"